android/app/src/main/java/org/fossasia/openevent/activities/BaseActivity.java

```diff
@@ -52,6 +52,8 @@ public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             case android.R.id.home:
                 onBackPressed();
+                break;
+            default:
```

Comment:
I'm confused what to write because in every switch statement case is action bar menu item. And we don't want to do anything if it is not menu item.

Will "Do nothing" be okay ?